cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -350,7 +350,7 @@ declare namespace Deno {
    *
    * @category FFI
    */
-  export type PointerValue = number | bigint;
+  export type PointerValue = null | Record<string | number | symbol, never>;
```

Comment:
Using an opaque type here would allow TypeScript to detect more errors.

```ts
const brand: unique symbol;
type External = { [brand]: unknown };
export type PointerValue = External | null;
```

```ts
// @ts-expect-error: Property '[brand]' is missing in type '{}' but required in type 'External'.
const ptr: NonNullable<Deno.PointerValue> = {};
// @ts-expect-error: Property 'prop' does not exist on type 'External'.
ptr.prop;
```

Reply:
Nice, thank you! I was pondering about setting up some custom `class External` or something but decided that would be wrong as someone might think they could use `instanceof External`.

This works wonders.